Hackney.Shared.Tenure/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
+
+ARG LBHPACKAGESTOKEN
+ENV LBHPACKAGESTOKEN=$LBHPACKAGESTOKEN
```

Comment:
This is only need if you're using the Hackney.Core packages.
So you can either remove it or 
* Undelete the NuGet.Config file and update it to look like one of the other Api projects.
* Add this line after the other COPY commands (l.9)
```COPY /NuGet.Config /root/.nuget/NuGet/NuGet.Config```
* Ensure the CircleCi pipeline has the LBHPACKAGESTOKEN env variable set.